src/test/kotlin/no/nav/eessi/pensjon/fagmodul/prefill/person/PrefillPDLNavTest.kt

```diff
@@ -308,7 +301,7 @@ class PrefillPDLNavTest {
         val prefillData = PrefillDataModelMother.initialPrefillDataModel(SedType.P2000, pinId = somePersonNr, penSaksnummer = somePenSaksnr)
         val personDataCollection = PersonDataCollection(forsikretPerson = person, ektefellePerson = partner, sivilstandstype = Sivilstandstype.REGISTRERT_PARTNER, gjenlevendeEllerAvdod = person, barnPersonList = emptyList())
 
-        doReturn("NO").`when`(kodeverkClient).finnLandkode2("NOR")
+        every { kodeverkClient.finnLandkode2("NOR") } returns "NO"
```

Comment:
Denne kan kanskje flyttes til @BeforeEach, i stedet for å ha den i hver test?

Reply:
Bra sett!